examples/petsc/setup.h

```diff
@@ -565,10 +565,10 @@ static int SetupLibceedByDegree(DM dm, Ceed ceed, CeedInt degree, CeedInt dim,
   CeedOperatorApply(op_setupgeo, xcoord, qdata, CEED_REQUEST_IMMEDIATE);
 
   // Set up RHS if needed
+  CeedVectorCreate(ceed, nelem*nqpts*ncompu, target);
```

Comment:
Moved?

Reply:
Yes, outside of the conditional block so that it may not be used uninitialized.

Reply:
Is there a case in which that occurred? The right hand side and target are only ever generated together.

Reply:
This might be creating memory leak in the multigrid example because it's creating this vector multiple times. I really don't think we want to do this.

Reply:
Okay, I'll change it back for the memory leak. I had change it when I was debugging my code and was calling the `SetupLibceedByDegree()` with `false`, so that was a problem because that vector was never initialized. By I agree that this is not the case, since we always call it with `true` in this problem.